File: C/arit.c
#include <stdio.h>

int main() {
    float num1, num2, soma, sub, mult, div;

    printf("Digite o primeiro número: ");
    scanf("%f", &num1);
    printf("Digite o segundo número: ");
    scanf("%f", &num2);

    soma = num1 + num2;
    sub = num1 - num2;
    mult = num1 * num2;
    div = num1 / num2;

    printf("Soma: %.2f\n", soma);
    printf("Subtração: %.2f\n", sub);
    printf("Multiplicação: %.2f\n", mult);
    printf("Divisão: %.2f\n", div);

    return 0;
}

File: C#/arit.cs
using System;

class Program {
    static void Main() {
        Console.Write("Digite o primeiro número: ");
        float num1 = float.Parse(Console.ReadLine());

        Console.Write("Digite o segundo número: ");
        float num2 = float.Parse(Console.ReadLine());

        float soma = num1 + num2;
        float sub = num1 - num2;
        float mult = num1 * num2;
        float div = num1 / num2;

        Console.WriteLine($"Soma: {soma}");
        Console.WriteLine($"Subtração: {sub}");
        Console.WriteLine($"Multiplicação: {mult}");
        Console.WriteLine($"Divisão: {div}");
    }
}
